README.md

```diff
@@ -7,14 +7,14 @@
 [![Discord Server](https://img.shields.io/discord/844600078504951838?color=7289da&logo=discord "Discord Server")](https://discord.gg/jTysUaxXzM)
 [![Twitter](https://img.shields.io/twitter/follow/authzed?color=%23179CF0&logo=twitter&style=flat-square "@authzed on Twitter")](https://twitter.com/authzed)
 
-SpiceDB is a [Zanzibar]-inspired database that stores, computes, and validates application permissions.
+SpiceDB is a database system for managing security-critical permissions checking.
```

Comment:
"Managing ... checking" reads oddly to me?

Reply:
>SpiceDB is a database system for managing security-critical application permissions.

how about this?